.spotbugs/spotbugs-exclude.xml

```diff
@@ -16,5 +31,10 @@
   <Match>
     <Class name="~io\.fabric8\.kubernetes\.api\.model\.(Doneable).+(\$.*)?" />
   </Match>
+  <Match>
+    <Class name="~io\.strimzi\.systemtest\..+(\$.*)?" />
+  </Match>
+
+  io.strimzi.systemtest
```

Comment:
Is this here intentionally kind of outside of the XML?

Reply:
Oops.